apps/condo/domains/acquiring/utils/freezeBillingReceipt.spec.js

```diff
@@ -1,3 +1,7 @@
+/**
+ * @jest-environment node
```

Comment:
At the moment all spec files use browser environments. I don't think that we need to change it. Because our utilities should work in different environments. The browser environment imposes more restrictions.